tools/mason/MasonBuild.chpl

```diff
@@ -0,0 +1,174 @@
+
+use TOML;
+use Spawn;
+use FileSystem;
+use MasonUtils;
+use MasonHelp;
+
+
+
+proc masonBuild(args) {
+  var show = false;
+  var release = false;
+  var compopts: [1..0] string;
+  if args.size > 2 {
+    for arg in args[2..] {
+      if arg == '-h' || arg == '--help' {
+	masonBuildHelp();
+	exit();
+      }
+      else if arg == '--release' {
+	release = true;
+      }
+      else if arg == '--show' {
+	show = true;
+      }
+      else {
+	compopts.push_back(arg);
+      } 
+    }
+  }
+  UpdateLock();
+  BuildProgram(release, show, compopts);
+}
+
+
+proc BuildProgram(release: bool, show: bool, compopts: [?d] string) {
+  if isFile("Mason.lock") {
+
+    // --fast
+    var binLoc = 'debug';
+    if release {
```

Comment:
For 1-line `if-then` statements, you can do:

```chpl
if release then binLoc = 'release';
```

or with a newline (as some prefer):

```chpl
if release then 
  binLoc = 'release';
```